src/formatDate.js

```diff
@@ -50,7 +50,31 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
+  const resultArr = [];
+  const tempDate = {};
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    tempDate[fromFormat[i]] = dateArr[i];
+  }
+
+  if (tempDate.hasOwnProperty('YYYY')) {
+    tempDate['YY'] = tempDate['YYYY'].toString().split('').slice(2).join('');
```

Comment:
```suggestion
    tempDate['YY'] = tempDate['YYYY'].slice(2);
```

Reply:
When I tried slice type of Number, it wasn't work. Now try again and it's work fine. Didn't understood why yet.